diff --git a/SDL_mixer.h b/SDL_mixer.h
index 71c1f5c..73b1f74 100644
--- a/SDL_mixer.h
+++ b/SDL_mixer.h
@@ -24,12 +24,12 @@
 #ifndef _SDL_MIXER_H
 #define _SDL_MIXER_H
 
-#include "SDL_stdinc.h"
-#include "SDL_rwops.h"
-#include "SDL_audio.h"
-#include "SDL_endian.h"
-#include "SDL_version.h"
-#include "begin_code.h"
+#include "SDL2/SDL_stdinc.h"
+#include "SDL2/SDL_rwops.h"
+#include "SDL2/SDL_audio.h"
+#include "SDL2/SDL_endian.h"
+#include "SDL2/SDL_version.h"
+#include "SDL2/begin_code.h"
 
 /* Set up for C function definitions, even when using C++ */
 #ifdef __cplusplus
@@ -626,6 +626,6 @@ extern DECLSPEC void SDLCALL Mix_CloseAudio(void);
 #ifdef __cplusplus
 }
 #endif
-#include "close_code.h"
+#include "SDL2/close_code.h"
 
 #endif /* _SDL_MIXER_H */
